IntelliTect.Multitool/ReleaseDateAttribute.cs

```diff
@@ -10,12 +10,12 @@ namespace IntelliTect.Multitool;
 public class ReleaseDateAttribute : Attribute
 {
     /// <summary> 
-    /// Constructor called from csproj file
+    /// Constructor called from .targets file
```

Comment:
Technically neither of these are correct. The targets file triggers some source generation, the just creates a C# file with an assembly attribute.